zio/csvio/ztests/bstring.yaml

```diff
@@ -12,12 +12,5 @@ input: |
 
 output-flags: -f csv
 
-output: |
-  bstring
-  -
-  
-  """"
-  ","
-  \x2d
-  \\
-  \x00\xff
+output: !!binary |
+  YnN0cmluZwoKCiIiIiIKIiwiCi0KXAoA/wo=
```

Comment:
The old output has a logical correspondence to the input, but the new output just checks that the encoding has not changed from build to build. How do we verify the !!binary string included in the output still has the same semantics if for some reason it changes?

Reply:
@marktwallace: The logical correspondence remains, though to see it in the YAML, you do unfortunately have to decode the Base64 string. But a test failure will print a diff against the decoded bytes, so there's that.

However, absent a ztest feature that interprets escape sequences in `output`, `!!binary` seems like the best way to include arbitrary bytes in it.